packages/loot-core/src/platform/client/fetch/index.browser.js

```diff
@@ -77,7 +77,7 @@ function handleMessage(msg) {
 function connectWorker(worker, onOpen, onError) {
   globalWorker = worker;
 
-  worker.addEventListener('message', event => {
+  worker.onmessage = event => {
```

Comment:
Does `addEventListener` not work here? Is there any possibility something else will try to set `onmessage`?

Reply:
With the move to webpack v5, we are no longer using `workers-loader`. This also _slightly_ changes the web-worker API interface. In the new webpack v5 web workers there is no `addEventListener`.

https://webpack.js.org/guides/web-workers/